src/FlaUI.Core/Elements/Tab.cs

```diff
@@ -5,49 +5,49 @@
 
 namespace FlaUI.Core.Elements
 {
-    public class Tab : Element
+    public class Tab : AutomationElement
     {
         public Tab(AutomationObjectBase automationObject) : base(automationObject)
         {
         }
 
         /// <summary>
-        /// The currently selected <see cref="TabItem"/>
+        /// The currently selected <see cref="TabItemAutomation"/>
         /// </summary>
-        public TabItem SelectedTabItem
+        public TabItemAutomation SelectedTabItemAutomation
```

Comment:
Switched all TabItemAutomation and TabItemsAutomation back to TabItem and TabItems respectively. Resharper renamed this and I didn't notice.